### PR TITLE
Adding code sample to switch

### PR DESCRIPTION
**Pull request by vuchl**
fixing #157

### CODE REVIEW

File: docs/src/App.vue [hunk 1]
```diff
@@ -62,6 +62,10 @@
                 </md-list-item>
 
                 <md-list-item class="md-inset">
```
Comment (marcosmoura) [inline docs/src/App.vue]: This is not supposed to enter in this PR.

File: docs/src/routes.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ import BottomBar from './pages/components/BottomBar';
 import Buttons from './pages/components/Buttons';
 import ButtonToggle from './pages/components/ButtonToggle';
 import Card from './pages/components/Card';
+import Chips from './pages/components/Chips';
```
Comment (marcosmoura) [inline docs/src/routes.js]: This is not supposed to enter in this PR.

File: src/index.js [hunk 3]
```diff
@@ -5,6 +5,7 @@ import MdBottomBar from './components/mdBottomBar';
 import MdButton from './components/mdButton';
 import MdButtonToggle from './components/mdButtonToggle';
 import MdCard from './components/mdCard';
+import MdChips from './components/mdChips';
```
Comment (marcosmoura) [inline src/index.js]: This is not supposed to enter in this PR.

Comment (vuchl) [inline src/index.js]: My push of the chips is also merged. I'll revert the `367eea8` commit so the repo only contains the sample for switch.

**Comment by vuchl** [comment]
I hope I did that correctly. Did a `git revert 367eea8` and a `git push` afterwards. Not the biggest git user.

**Pull request merged**